PVEntityGenerator/xml/PVEntityGenerator.xml

```diff
@@ -7,6 +10,7 @@
     <platform name="Java2" definition-dir="Platform_Java2"/>
     <platform name="Java5" definition-dir="Platform_Java5"/>
     <platform name="Java5JPA" definition-dir="Platform_Java5JPA"/>
+	  <platform name="Java11" definition-dir="Platform_Java11"/>
```

Comment:
is this new platform derived from Platform_Java5 or Platform_Java5JPA? in case of the latter, i would prefer a name like "Platform_Java11JPA" to reflect this.

cosmetic: please fix indentation.